biggraphite/drivers/elasticsearch.py

```diff
@@ -119,6 +121,12 @@
 DEFAULT_PORT = 9200
 DEFAULT_TIMEOUT = 10
 
+MINUTE = 60
```

Comment:
This seems copied from cassandra.py, could be worth putting them in some common files

Reply:
Done